### PR TITLE
shellscript: no auto cwd option

### PR DESCRIPTION
**Pull request by bryanmacfarlane**

### CODE REVIEW

File: Tasks/ShellScript/task.json [hunk 1]
```diff
@@ -53,6 +53,15 @@
             "groupName": "advanced"
         },
         {
+            "name": "disableAutoCwd",
+            "type": "boolean",
+            "label": "Disable Auto CWD",
+            "defaultValue": "false",
```
Comment (ericsciple) [inline Tasks/ShellScript/task.json]: Consider tying the visibility of the cwd input to this flag. If the checkbox is not checked (default behavior) the working directory input shouldn't be visible at all.

Comment (ericsciple) [inline Tasks/ShellScript/task.json]: ...if you tie to the visibility, you might also want to put the disableAutoCwd checkbox above the cwd input.

I believe this rule on the cwd input would suffice:
```
"visibleRule": "disableAutoCwd = true",
```

Comment (bryanmacfarlane) [inline Tasks/ShellScript/task.json]: Good suggestion ... But, I'm trying to decide whether cwd should be promoted out of advanced and if this should be left in advanced.  In that case we wouldn't want conditioned.

Comment (ericsciple) [inline Tasks/ShellScript/task.json]: Nevermind, it's a sticky situation... the checkbox only kicks in if !filePathSupplied. Visibility rule would just make matters worse.

Comment (ericsciple) [inline Tasks/ShellScript/task.json]: Looks good.

**Pull request merged**